data/fields/castle_type.json

```
@@ -1,5 +1,18 @@
 {
     "key": "castle_type",
     "type": "combo",
-    "label": "Type"
+    "label": "Type",
+    "strings": {
+        "options": {
+            "defensive": "Defensive castle",
+            "palace": "Palace",
+            "stately": "Stately home",
+            "manor": "Manor house",
+            "fortress": "Fortress",
+            "castrum": "Roman fort",
```

the convention is to use [title case](https://en.wikipedia.org/wiki/Title_case) for labels. There are some exceptions here and there, but I would say that we should write it like this here:

```suggestion
            "defensive": "Defensive Castle",
            "palace": "Palace",
            "stately": "Stately Home",
            "manor": "Manor House",
            "fortress": "Fortress",
            "castrum": "Roman Fort",
```